### PR TITLE
update prisma web

### PR DESCRIPTION
**Pull request by invpuppet**
all english, norm, addition to container

### CODE REVIEW

**Review by tbellicha — COMMENTED** [review]
Did you check the fields (if needed) in the code as well ?

**Comment by invpuppet** [comment]
I checked before push but if i left some don't hesitate to tell me

**Comment by tbellicha** [comment]
I meant, since you changed some names in the schema, did you check all the possible usages of these elemnts ?

**Comment by invpuppet** [comment]
my bad i left some data that were not with the good names

File: Configurateur/back/routes/contact.js [hunk 1]
```diff
@@ -4,13 +4,13 @@ const contactCtrl = require('../controllers/contact')
 
 router.post('/contact', async function (req, res, next) {
   try {
-    const { firstName, lastName, email, message } = req.body
+    const { firsName, lastName, email, message } = req.body
```
Comment (tbellicha) [inline Configurateur/back/routes/contact.js:7]: firsName ?

File: Configurateur/back/routes/contact.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ const contactCtrl = require('../controllers/contact')
...
     if (!email) {
       res.status(400)
       throw new Error('Email is required')
     }
   
-    const msg = await contactCtrl.registerMessage({ lastName, firstName, email, message })
+    const msg = await contactCtrl.registerMessage({ lastName, firsName, email, message })
```
Comment (tbellicha) [inline Configurateur/back/routes/contact.js:13]: firsName ?

**Review by tbellicha — CHANGES_REQUESTED** [review]

**Review by tbellicha — APPROVED** [review]

**Pull request merged**